exercism/bowling/bowling.js

```diff
@@ -0,0 +1,88 @@
+const Bowling = function(rolls) {
+  this.rolls = rolls
+
```

Comment:
I feel like adding some sort of "rollCountPerFrame" field to this class would simplify a lot of the logic when it comes to checking if you are allowed to roll or not.

I like to spend time thinking of what the class I'm making **IS** and what it can **DO**.
The **IS** would be its fields and the **DO** would be its methods.